README.md

```diff
@@ -9,7 +9,9 @@ $ faas new --list
 
 Languages available as templates:
 - golang-http
+- golang-http-armhf
 - golang-middleware
+- golang-middleware-armhf
```

Comment:
Please add a short sentence or paragraph explaining which to use for ARMv7 or Raspberry Pi.

Reply:
Got it :+1:

Reply:
Addressed it